### ISSUE TITLE
Using Selenium @FindBy with @ByAngularXYZ.FindBy

### ISSUE THREAD

**Issue by kathyrollo** [paul-hammant/ngWebDriver#60]
Hello Paul,
Relating to #26, is it possible to have a similar implementation as Selenium's native FindBy?

`@FindBy(how = How.BY_ANGULAR_MODEL, rootSelector = "butter", model = "cheese")`

Same with FindBys and FindAll.

I am trying to use Selenium's FindBy with the ngWebDriver FindBy [here](https://github.com/paul-hammant/ngWebDriver/blob/master/src/test/java/com/paulhammant/ngwebdriver/FindByNgWebDriverTest.java) but they do not appear to work together when used in the same page object.

**Comment by paul-hammant** [comment]
Does:

```
@ByAngularModel.FindBy(rootSelector = "butter", model = "cheese")
```
Do it for you?

**Comment by kathyrollo** [comment]
Hi Paul,
So far I have to use the fully qualified class name if I want to use with Selenium's FindBy in the same page object:
```
import org.openqa.selenium.support.FindBy;

public class HomePage {
    @com.paulhammant.ngwebdriver.ByAngularModel.FindBy(rootSelector = "butter", model = "cheese")
    private WebElement elementByModel;

    @FindBy(how = How.ID, using = "id")
    private WebElement elementById;
}
```
Any tips on how this may be achieved?

**Comment by paul-hammant** [comment]
You sure you cannot do it like this:

```
import com.paulhammant.ngwebdriver.ByAngularBinding;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.FindBy;
import org.openqa.selenium.support.How;

public class Bar {

    @ByAngularBinding.FindBy(rootSelector = "butter", binding = "cheese")
    public WebElement cheese;

    @FindBy(how = How.ID, using = "id")
    public WebElement how;
    
}
```

**Comment by kathyrollo** [comment]
Hi Paul,
The imports are working together now, thanks for showing how. :+1: 

Can ngWebDriver support FindBys and FindAll in the same way?
`@ByAngularBinding.FindAll`

**Comment by paul-hammant** [comment]
Broadening the issue - tsk tsk!

Hi @kathyrollo It is about 45 mins work for someone who's comfortable with JUnit:

https://github.com/paul-hammant/ngWebDriver/blob/master/src/test/java/com/paulhammant/ngwebdriver/FindByNgWebDriverTest.java

Pull requests accepted :)

**Comment by bhasksaha** [comment]
Hi @paul-hammant , can we initialize the page objects(elements we identified using @ByAngularBinding) in ngWebdriver like we do in Pagefactory

Eg: PageFactory.initElements(driver, this);

Thanks in advance !

**Comment by Rohitm7** [comment]
Hi paul,
I am new in ngwebdriver ,i want to use ngwebdriver locators in page factory model where i can learn all types..please help me

**Comment by paul-hammant** [comment]
I can’t help I’m sorry. Is there a dev on your team that can?

**Comment by Moustafa-Sherif** [comment]
Hi @paul-hammant 
I'm working on a page-object-model project where I want to handle a dropdown using ng-select, and I ran out of my methods as initially I was getting "Element should have been select but is ng-select", then when I used NgWebDriver I had "JavaScript error: Angular not defined". So, I don't know if I'm being mistaken with the locator or what. I would be greatly appreciated if there was any workaround I could implement.

![ng-select](https://user-images.githubusercontent.com/107168536/196740021-0728be13-b068-4877-9899-208056de267e.PNG)
